CHANGELOG.md

```diff
@@ -5,6 +5,54 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Register this Doorkeeper extension as `device_code` custom OAuth Grant Flow.
+
+### Changed
+- Upgrade `doorkeeper` dependency, matching versions `~> 5.5`.
+- Use the standard IANA URN value `urn:ietf:params:oauth:grant-type:device_code`
+  as grant type for device access token requests. It replaces the previous
+  value `device_code`, which was deliberately nonstandard to make it work with
+  Doorkeeper versions prior to `5.5`.
+
+  The old `device_code` value might still work out of the box with some
+  Doorkeeper releases, thanks to a fallback strategy; however, a warning message
+  will be printed, and this functionality will be eventually removed in newer
+  Doorkeeper versions.
```

Comment:
"might still work" sound a little too ambitious to me.

I assume that this is compatible with at least Doorkeeper 5.5.1, so we could say:

```suggestion
  The old `device_code` value still works out of the box with Doorkeeper <= 5.5.1. However, a warning message
  will be printed, and this functionality will be eventually removed in newer Doorkeeper versions.
```